src/main/java/com/javabom/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,31 @@
+package com.javabom.lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoNumber {
```

Comment:
```suggestion
public class LottoNumbers {
```
뒤에 s를 붙이는 편이 더 좋아보입니다. 처음 봤을 때 이 객체가 List를 들고있다는 생각을 하지 못했어요.